unlock-app/src/components/interface/buttons/Button.js

```diff
@@ -1,71 +1,74 @@
-import { Link } from 'react-router-dom'
```

Comment:
Noticed your last commit after rebase to modify Button props during rebase, I kept my changes in since it affected `Link`, let me know how you would like to keep the API here.

Reply:
So, I am not sure how next-router works, but the reason I replaced the links to use `Link` from react router was to make sure we benefitted from client side nav and not fully reload the whole page when a link is clicked. 

Is this similar with next-router?

Reply:
Yes, using Link will enable client side nav but it is recommended to add the anchor tag to pass the href to it (which I added here by default here if we need to use the button for external links like the Github icons).